### PR TITLE
CEDS-4960 On CYA page, shift from CSS Table to CSS list for section 'Authorisation holders'

### PR DESCRIPTION
**Pull request by udinobi**

### CODE REVIEW

File: app/views/declaration/summary/additional_documents.scala.html [hunk 1]
```diff
@@ -77,7 +77,7 @@
     else Some(SummaryListRow(
         classes = s"additional-documents-${item.sequenceId}-row",
         key = Key(content = Text(messages("declaration.summary.items.item.additionalDocuments"))),
-        value = Value(content = Text(messages("declaration.summary.items.item.additionalDocuments.none"))),
```
Comment (glidester) [inline app/views/declaration/summary/additional_documents.scala.html:80]: Can these deprecated message keys be removed from the messages file?

File: app/views/declaration/summary/sections/countries_section.scala.html [hunk 2]
```diff
@@ -45,8 +45,8 @@
                 ),
                 value = Value(
                     content = Text(
-                        if(declarationData.locations.routingCountries.isEmpty){
-                            messages("declaration.summary.countries.routingCountries.none")
```
Comment (glidester) [inline app/views/declaration/summary/sections/countries_section.scala.html:49]: Can these deprecated message keys be removed from the messages file?

**Review by glidester — APPROVED** [review]
Two nits :+1:

File: app/views/declaration/summary/additional_documents.scala.html (hunk 1 above)
Comment (udinobi) [reply app/views/declaration/summary/additional_documents.scala.html:80]: Did not I already remove them?

File: app/views/declaration/summary/sections/countries_section.scala.html (hunk 2 above)
Comment (udinobi) [reply app/views/declaration/summary/sections/countries_section.scala.html:49]: Did not I already remove them?

File: app/views/declaration/summary/additional_documents.scala.html (hunk 1 above)
Comment (glidester) [reply app/views/declaration/summary/additional_documents.scala.html:80]: Yes you did. My bad :+1:

**Pull request merged**